src/DependencyInjection/Compiler/AdminMakerCompilerPass.php

```diff
@@ -21,13 +23,13 @@
  */
 final class AdminMakerCompilerPass implements CompilerPassInterface
 {
-    const MANAGERS = [
+    public const MANAGERS = [
```

Comment:
this can be private?

Reply:
Yes it should be private 👍🏻

Reply:
Or can we get it from the configuration somehow?

Reply:
You can see it here: https://travis-ci.org/sonata-project/SonataAdminBundle/jobs/438008151